src/enum.cr

```diff
@@ -274,9 +274,8 @@ struct Enum
     value == other.value
   end
 
-  # Returns a hash value. This is the hash of the underlying value.
-  def hash
-    value.hash
+  def hash(hasher)
```

Comment:
Missing doc

Reply:
I'll add a "See `Object#hash(hasher)` to all of these